syntax = "proto2";
package colony.interface.proto;

import "util/proto/object_id.proto";

message SetProduction {
  optional util.proto.ObjectId area_id = 1;
  optional uint64 field_idx = 2;
  optional string process_name = 3;
}

message PlayerAction {
  oneof action {
    SetProduction set_production = 1;
  }
}

